### PR TITLE
Showcase Additions

### PR DESCRIPTION
**Pull request by marcshilling**
Added the Google Play link for DiscoveryVR.
Added a new iOS React Native app - wego concerts

### CODE REVIEW

**Comment by ide** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
@ide, be careful accepting PRs, this one broke master branch https://circleci.com/gh/facebook/react-native/4910?utm_campaign=build-failed&utm_medium=email&utm_source=notification

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ var featured = [
   {
     name: 'Discovery VR',
     icon: 'http://a2.mzstatic.com/us/r30/Purple6/v4/d1/d5/f4/d1d5f437-9f6b-b5aa-5fe7-47bd19f934bf/icon175x175.png',
-    link:  'https://itunes.apple.com/us/app/discovery-vr/id1030815031?mt=8',
+    linkAppStore: 'https://itunes.apple.com/us/app/discovery-vr/id1030815031?mt=8',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.discovery.DiscoveryVR'
```
Comment (bestander) [inline website/src/react-native/showcase.js]: comma is missing

**Comment by bestander** [comment]
not a big break, it won't disrupt anyone but CI is red

**Pull request closed without merging**